src/Core/Models/Imports/TetraCubeCreature.cs

```diff
@@ -0,0 +1,106 @@
+        public string Type { get; set; }
+        public string Tag { get; set; }
+        public string Alignment { get; set; }
+        public int HitDice { get; set; }
+        public string ArmorName { get; set; }
+        public int ShieldBonus { get; set; }
+        public int NatArmorBonus { get; set; }
+        public string OtherArmorDesc { get; set; }
+        public int Speed { get; set; }
+        public int BurrowSpeed { get; set; }
+        public int ClimbSpeed { get; set; }
+        public int FlySpeed { get; set; }
+        public bool Hover { get; set; }
+        public int SwimSpeed { get; set; }
+        public bool CustomHP { get; set; }
+        public bool CustomSpeed { get; set; }
+        public string HpText { get; set; }
+        public string SpeedDesc { get; set; }
+        public int StrPoints { get; set; }
+        public int DexPoints { get; set; }
+        public int DexModifier
+        {
+            get { return (DexPoints - 10) / 2; }
+            set { DexPoints = value / 2; }
+        }
+        public int ConPoints { get; set; }
+        public int IntPoints { get; set; }
+        public int WisPoints { get; set; }
+        public int ChaPoints { get; set; }
+        public int Blindsight { get; set; }
+        public bool Blind { get; set; }
+        public int Darkvision { get; set; }
+        public int Tremorsense { get; set; }
+        public int Truesight { get; set; }
+        public int Telepathy { get; set; }
+        [JsonPropertyName("cr")]
+        public string Cr { get; set; }
+        public string CustomCr { get; set; }
+        public int CustomProf { get; set; }
+        public bool IsLegendary { get; set; }
+        public string LegendariesDescription { get; set; }
+        public bool IsLair { get; set; }
+        public string LairDescription { get; set; }
+        public string LairDescriptionEnd { get; set; }
+        public bool IsMythic { get; set; }
+        public string MythicDescription { get; set; }
+        public bool IsRegional { get; set; }
+        public string RegionalDescription { get; set; }
+        public string RegionalDescriptionEnd { get; set; }
+        public int ArmorClass { get; set; }
+
+        public ICollection<Ability> Abilities { get; set; } = Array.Empty<Ability>();
+        public ICollection<Action> Actions { get; set; } = Array.Empty<Action>();
+
+        public ICollection<Skill> Skills { get; set; } = Array.Empty<Skill>();
+
+        public string UnderstandsBut { get; set; }
+        public string ShortName { get; set; }
+        public string PluralName { get; set; }
+        public bool DoubleColumns { get; set; }
+        public int SeparationPoint { get; set; }
+
+    }
+    public static class TetraCubeCreatureExtensions
+    {
+        public static int CalculateACFromTetraCube(this TetraCubeCreature creature)
+        {
+            var modifierWithCap = creature.DexModifier;
+            if (modifierWithCap > 2)
```

Comment:
Regarding the 5e calc from above I think this is the same thing.

Reply:
hmm I agree with the other one, not sure if I agree that modifer is a core calculation or calculate from tetracube is

Reply:
Isn’t this the medium armor maximum?